bossdata/remote.py

```diff
@@ -197,14 +200,30 @@ def get(self, remote_path, progress_min_size=10, auto_download=True):
                 then a RuntimeError occurs.
 
         Returns:
-            str: Absolute local path of the local file that mirrors the remote file.
+            str, int: Absolute local path of the local file that mirrors the remote file, and
```

Comment:
Adding the index to the return value forces all API users to handle it, even though its usually an implementation detail they don't care about.  It also breaks all code that uses `get` that has not yet been updated (e.g., `bossfetch`).  Instead, we can let the caller that actually cares about this use something like:
```
local_path = mirror.get(remote_paths)
if local_path != remote_paths[0]:
    print('{} was substituted for {}.'.format(local_path, remote_paths[0]))
```
This pattern should be documented in the docstring for `get`.